game.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'environment'
+
+class Game 
+  attr_accessor :depot, :control
+  include Environment
+
+  def initialize
+    @board = { payers: {}, control: {}, status: nil}
+    @depot = Environment::Depot.new
+    @control = Environment::Control.new
+  end
+
+  def new_worker(name:)
+    worker = Environment::Worker.new(name:name, personal_data: nil,  professional_data: nil)
+    @board[:payers[name]] = worker
+    
+  end
+
+  def new_mission(name:, objective: , pack: )
+
+    @control.new_mission(name:name, objective:objective, pack:pack)
```

Comment:
Same question here, did not you have trouble with `objective:objective` and `pack:pack`? I usually insert a space after the `:`

Reply:
Yes, It didn't show me any warning or error